python/hsfs/constructor/query.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         self._storage_connector_api = storage_connector_api.StorageConnectorApi(
             feature_store_id
         )
+        self._is_time_travel = False
```

Comment:
For the sake of consistency, we should set _is_time_travel according to whether "left_feature_group_start_time" and "left_feature_group_start_time" are None/non None.